aws.yaml

```diff
@@ -7,7 +7,7 @@
       version: 1.2.1
     - app: cert-manager
       componentVersion: 0.9.0
-      version: 1.0.5
+      version: 1.0.6
```

Comment:
@paurosello I've bumped cert-manager too in this PR, taken over from 11.0.2 release definition.

Reply:
thanks!